src/export-api-v2.ts

```diff
@@ -0,0 +1,615 @@
+import path from 'path'
+import fs from 'fs'
+type FetchResponse = Response
+/** CONFIG / CONSTANTS */
+const DCL_SUBGRAPH_API_KEY = process.env.DCL_SUBGRAPH_API_KEY || 'b53c670e2e03f823f46ab1b2087835c6'
+
+const RENTALS_SUBGRAPH_URL = 'https://subgraph.decentraland.org/rentals-ethereum-mainnet'
+const MARKETPLACE_SUBGRAPH_URL = 'https://subgraph.decentraland.org/marketplace'
+const COLLECTIONS_POLYGON_SUBGRAPH_URL = 'https://subgraph.decentraland.org/collections-matic-mainnet'
+
+const DCL_RENTALS_FEE_PCT = 0.025
+const PRIMARY_SALES_FEE_PCT = 0.025
+const NAME_CONTRACT = '0x2a187453064356c898cae034eaed119e1663acb8'.toLowerCase()
+
+const ALCHEMY_ETH_URL = process.env.ALCHEMY_ETH_URL || 'https://eth-mainnet.g.alchemy.com/v2/25n756T7F5Lfxjx34PSfX'
```

Comment:
```suggestion
const ALCHEMY_ETH_URL = process.env.ALCHEMY_ETH_URL || 'https://eth-mainnet.g.alchemy.com/v2/25n756T7F5Lfxjx34PSfX'
```

is this a api_key or something ? or its public ?